src/js/components/LoginPage/Login.js

```diff
@@ -1,11 +1,100 @@
 import React, { Component } from "react";
 import "./Login.scss";
 import "./ButtonStyles.css";
+import Navbar from "../Navbar/Navbar";
+import {Redirect} from "react-router-dom";
+import StyledFirebaseAuth from "../RegisterPage/SignUp";
+import {RegisterUser} from "../../services/apiservice";
+import { BrowserRouter, Route, Switch, NavLink } from "react-router-dom";
+
+
```

Comment:
Although the reviewers are told to not point whitespaces as its not as big issue but it does not mean you'll leave so many whitespaces... if it was not so many I would not point it out as I left many PRs with whitespaces now.
Line 10 and 21 of this file has whitespaces as well as gc on line 24. 
There is no space between line 45 and 46. 
Line 51 has gc as well. 
The file *Login.scss* also has whitespaces on line 101 and 102 
even thou it was addressed not to use 
`!important` 
instead of removing it, its commented out.

Reply:
Thank you fixed it